examples/aides_logement/tests/tests_calcul_al_accession_propriete.catala_fr

```diff
@@ -75,24 +75,26 @@ champ d'application Exemple2:
 ```catala-test-inline
 $ catala Interpret -s Exemple1
 [RESULT] Computation successful! Results:
-[RESULT] montant = 96.48 €
+[RESULT] montant = ¤96.48
```

Comment:
Not OK

Reply:
There was quite a bit of pondering for this change, so let me explain my reasoning before insisting that you disagree :D

- We have a printer for expressions in the different variants of the AST ; that printer does not correspond to the Catala syntax, and is not localised ; we use it mainly for debugging.
- There was a single spot where that printer actually relied on a global reference, affecting only the printing of money literals, to guess what unit the money should be printed as.

My point actually is — I've met this issue already with the experimental explanations plugin — that this printer isn't suitable for printing end-user results and that we should have a separate (probably fully localised) one ; that way we could have a consistent french syntax (`10,25 €` with a comma) or english syntax ($10.25 with a dot), among other things. (note that for results, this could be limited to literals hence fairly quick to write. for the explanation backend where we print formulas it'll be much more work, but I'm convainced that will be necessary at some point)

So for the sake of consistency I feel we need to use an undeterminate money symbol here. That's exactly what the unicode character `¤` is  :). It also means we get consistent output for a given program, not depending on configuration.

Of course, I fully understand that this symbol is little known and that users will be confused by this output ;) . The reason I am pushing this already is because the refactoring required that I got rid of the corresponding global reference. I can write a localised literal printer asap to recover better `[RESULT]` printing :)

Reply:
>  I can write a localised literal printer asap to recover better [RESULT] printing :)

OK for deleting the global reference but I would argue the localised literal printer should be in this PR (with a local optional parameter that defaults to English), I don't want to have a temporary state of Catala with `¤` everywhere :)

Reply:
Fair enough :wink: